ingenialink/dictionary.py

```diff
@@ -507,6 +533,24 @@ class DictionaryV3(Dictionary):
     __PDO_ENTRY_SIZE_ATTR = "size"
     __PDO_ENTRY_SUBNODE_ATTR = "subnode"
 
+    @classmethod
+    def get_description(cls, dictionary_path: str, interface: Interface) -> DictionaryDescriptor:
+        try:
+            with open(dictionary_path, "r", encoding="utf-8") as xdf_file:
+                tree = ET.parse(xdf_file)
+        except FileNotFoundError:
+            raise FileNotFoundError(f"There is not any xdf file in the path: {dictionary_path}")
+        root = tree.getroot()
+        device_path = (
+            f"{cls.__BODY_ELEMENT}/{cls.__DEVICES_ELEMENT}/{cls.__DEVICE_ELEMENT[interface]}"
+        )
+        device = cls.__find_and_check(root, device_path)
```

Comment:
If a dictionary does not have the requested interface this will raise an exception. Should it be more explicit?

Reply:
Same as V2: 
raise ILDictionaryParseError("Dictionary cannot be used for the chosen communication")
https://github.com/ingeniamc/ingenialink-python/pull/502/files